File: IntelligenceTraffic_Scala_v2.0/src/main/scala/com/uek/bigdata/DAO/inter/ITaskDao.scala
package com.uek.bigdata.DAO.inter

import com.uek.bigdata.daomain.Task

/**
  * 类名称: ITaskDao
  * 类描述:
  *
  * @Time 2018/11/29 1:09
  * @author 武建谦
  */
trait ITaskDao {
  //根据task 主键查询制定的任务数据
  def selectTaskById(taskId:Long): Task
}

File: IntelligenceTraffic_v3.0/src/main/java/com/uek/bigdata/DAO/inter/ITaskDao.java
package com.uek.bigdata.DAO.inter;

import com.uek.bigdata.daomain.Task;

/**
 * 类名称：ITaskDao
 * 类描述：任务管理dao层接口
 * @author 武建谦
 * @Time 2018/11/25 14:16
 */
public interface ITaskDao {
    //根据task 主键查询制定的任务数据
    public Task selectTaskById(long taskId);
}
